src/gwt/src/org/rstudio/studio/client/common/dependencies/DependencyManager.java

```diff
@@ -213,11 +213,11 @@ public void withRMarkdown(String progressCaption, String userAction,
       deps.add(Dependency.cranPackage("htmltools", "0.3.5"));
       deps.add(Dependency.cranPackage("caTools", "1.14"));
       deps.add(Dependency.cranPackage("bitops", "1.0-6"));
-      deps.add(Dependency.cranPackage("knitr", "1.14", true));
+      deps.add(Dependency.cranPackage("knitr", "1.14"));
```

Comment:
@jmcphers Is correct that we did that because of delays in CRAN binaries being available. That said, I think that any package which doesn't have native code benefits from the source install (as it allows it to install seamlessly in more versions of R). I'd leave this as-is but if you feel strongly that we want it to go binary we can try it and see if we get more support requests about failed rmarkdown installs or not :-)